lotto.py

```
@@ -0,0 +1,692 @@
+        latest_round = await conn.fetchrow(
+            """
+            SELECT id, status, merkle_root, entry_count, drand_round_target, anchor_hash,
+                   operator_pubkey, operator_signature, beacon, winner_idx, winner_commitment,
+                   claim_deadline, closed_at, drawn_at, claimed_at
+            FROM lotto_rounds
+            ORDER BY id DESC
+            LIMIT 1
+            """
+        )
+
+    start_date = date.today()
```

Ruff (0.15.9)

[warning] `datetime.date.today()` used

(DTZ011)